### PR TITLE
Fix ordering on page

### PR DESCRIPTION
**Pull request by jochenklar**
## Description

This PR fixes the worst bug in `2.0.0` so far. The questions (or questionsets) are not ordered correctly. Someone ... forgot to adjust the `PageSerializer` in `projects` to the new data model.

@MyPyDavid can you check if it works now?

### CODE REVIEW

**Comment by MyPyDavid** [comment]
I also see the bug now 😝 
> If re-used, who created the dataset?
> Is the dataset being created or re-used?

I'll check it with this PR installed..

**Comment by MyPyDavid** [comment]
a fix is nice... but a test for it would be better or not?

File: rdmo/projects/tests/test_viewset_project_page.py [hunk 1]
```diff
@@ -29,7 +29,10 @@
 
 projects = [1, 2, 3, 4, 5]
 pages = [1]
-
+ordered_pages = [16]
+odered_page_question_ids = {
```
Comment (jochenklar) [inline rdmo/projects/tests/test_viewset_project_page.py:33]: typo

File: rdmo/projects/tests/test_viewset_project_page.py [hunk 2]
```diff
@@ -45,3 +48,24 @@ def test_detail(db, client, username, password, project_id, page_id):
         assert response.json().get('id') == page_id
     else:
         assert response.status_code == 404
+
+
+@pytest.mark.parametrize('username,password', users)
```
Comment (jochenklar) [inline rdmo/projects/tests/test_viewset_project_page.py:53]: not for every user and project, please.

Comment (MyPyDavid) [reply rdmo/projects/tests/test_viewset_project_page.py]: Ive used now only the first with `[0:1]`, is ok ?

Comment (jochenklar) [reply rdmo/projects/tests/test_viewset_project_page.py]: why not remove the parametrization for users and projects and use `user` and `1` explicitly?

Comment (MyPyDavid) [reply rdmo/projects/tests/test_viewset_project_page.py]: is also possible.. hard coded tests look so strange, they are all parametrized..

Comment (MyPyDavid) [reply rdmo/projects/tests/test_viewset_project_page.py]: now its there just: `test_detail_order_in_page(db, client)`

**Comment by jochenklar** [comment]
:pray:

**Review by MyPyDavid — APPROVED** [review]

**Comment by MyPyDavid** [comment]
why is the coverall still pending? I thought it would go up with the last added test

**Pull request merged**